src/main/scala/uk/gov/nationalarchives/db/users/Lambda.scala

```diff
@@ -0,0 +1,31 @@
+
+package uk.gov.nationalarchives.db.users
+
+import scalikejdbc._
+import uk.gov.nationalarchives.db.users.Config._
+
+import java.io.{InputStream, OutputStream}
+import java.nio.charset.Charset
+
+class Lambda {
+
+  def process(inputStream: InputStream, outputStream: OutputStream) = {
+    println("Running")
+    val apiUser = createUser("consignment_api_user")
```

Comment:
Is it worth having the two user names in as config instead of hardcoded?

Reply:
I've hard coded them into the application.conf file and referenced it from there. There's not much point in having it as an environment variable because that just allows us to change the database username which I can't see happening.